storage/src/state/tests.rs

```diff
@@ -347,6 +348,32 @@ fn test_transaction_fees() {
     assert_eq!(output_record.value(), amount);
 }
 
+#[test]
+fn test_get_blocks_iterator() {
+    // Initialize a new temporary directory.
+    let directory = temp_dir();
+
+    // Initialize an empty ledger.
+    let ledger_state = LedgerState::open_writer::<RocksDB, _>(directory.clone()).expect("Failed to initialize ledger");
+
+    // Read the test blocks;
+    // note: they don't include the genesis block, as it's always available when creating a ledger.
+    let test_blocks = fs::read("benches/blocks_1").expect("Missing the test blocks file");
+    let blocks: Vec<Block<Testnet2>> = bincode::deserialize(&test_blocks).expect("Failed to deserialize a block dump");
+
+    // Load a test block into the ledger.
+    ledger_state.add_next_block(&blocks[0]).expect("Failed to add a test block");
+    let blocks_result: Vec<_> = ledger_state
+        .get_blocks(0, ledger_state.latest_block_height() + 1)
+        .unwrap()
+        .filter_map(|block_result| block_result.ok())
+        .collect();
+
+    drop(ledger_state);
```

Comment:
```suggestion
```
no need

Reply:
Done! The change was made in this commit [f4eccff](https://github.com/AleoHQ/snarkOS/pull/1753/commits/f4eccff4fbc5864b1ce3aeb3d519f1edf79bad88)